tests/foreman/api/test_subnet.py

```diff
@@ -46,64 +79,107 @@ def test_positive_add_parameter(self):
 
         :expectedresults: The parameter should be created in subnet
         """
+        subnet = entities.Subnet().create()
+        for name in generate_strings_list(exclude_types=['html']):
```

Comment:
very quick question: why html type is excluded?

Reply:
Because, I really dont thing that customer is going to use that kind of naming. Also how customer will use that parameter under inherited host with such kinda name is the question.

Reply:
cool, we are not guessing here, so if application permits that type of data, please test it out. One of the reason to use automation is possibility to test even almost not possible scenarios as it costs nothing for us

Reply:
Ok, I added html now.

Tests where I was asking to exclude 'html' are now passing including html.
```
# py.test tests/foreman/api/test_subnet.py -k test_positive_add_parameter
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.11, pytest-3.2.3, py-1.4.34, pluggy-0.4.0
rootdir: /home/jitendrayejare2/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.3, cov-2.4.0
collected 18 items                                                                                                                                                                                                 
2017-11-24 18:46:35 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/api/test_subnet.py ..

=============================================================================================== 16 tests deselected ===============================================================================================
==================================================================================== 2 passed, 16 deselected in 12.27 seconds =====================================================================================
# py.test tests/foreman/api/test_subnet.py -k test_positive_add_parameter_with_values_and_separator
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.11, pytest-3.2.3, py-1.4.34, pluggy-0.4.0
rootdir: /home/jitendrayejare2/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.3, cov-2.4.0
collected 18 items                                                                                                                                                                                                 
2017-11-24 18:47:50 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/api/test_subnet.py .

=============================================================================================== 17 tests deselected ===============================================================================================
===================================================================================== 1 passed, 17 deselected in 2.41 seconds =====================================================================================
# py.test tests/foreman/api/test_subnet.py -k test_positive_create_with_parameter_and_valid_separator
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.11, pytest-3.2.3, py-1.4.34, pluggy-0.4.0
rootdir: /home/jitendrayejare2/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.3, cov-2.4.0
collected 18 items                                                                                                                                                                                                 
2017-11-24 18:48:28 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/api/test_subnet.py .

=============================================================================================== 17 tests deselected ===============================================================================================
===================================================================================== 1 passed, 17 deselected in 6.64 seconds =====================================================================================
```